uqcsbot/member_counter.py

```diff
@@ -0,0 +1,79 @@
+import logging
+from datetime import datetime, timedelta
+import os
+import asyncio
+
+import discord
+from discord import app_commands
+from discord.ext import commands
+
+
+class MemberCounter(commands.Cog):
+    # Checks for an Azure specific environment variable, if it exists we're running as prod.
+    MEMBER_COUNT_CHANNEL_ID = 1094584746207617064 if os.environ.get("WEBSITE_SITE_NAME") != None \
```

Comment:
We want to not use IDs as much as possible, because they're opaque to any readers and require funky hacks like this to remain available in other servers (consider: the bot is locally hostable by anyone in any server, not just uqcs & testing). Is it possible to just get all channels and filter by "name starts with Member Count" ?

This is the one case where IDs are justifiable, due to name changes, so don't worry about it too much.

Reply:
Modified to use the beginning of the channel name to determine the right channel.

On a related note, should `SERVER_ID` within `bot.py` be changed as well? Do we guarantee that uqcsbot is only in one server (as if this is not true, grabbing channels by name is dubious).

Reply:
uhhhhhhhhhhhhhhhhh good question

Reply:
alright I may consistent-ise that shortly but for now if you just use the standard approach here (get_all_channels() and filter) then that should be fine